package.json

```diff
@@ -4,27 +4,32 @@
   "private": true,
   "proxy": "http://localhost:8080",
   "dependencies": {
-    "date-fns": "latest",
     "@date-io/date-fns": "v1",
-    "@material-ui/core": "^4.9.10",
+    "@material-ui/core": "^4.9.11",
     "@material-ui/icons": "^4.9.1",
-    "@material-ui/pickers": "latest",
+    "@material-ui/pickers": "3.2.10",
```

Comment:
Can you validate that the date pickers still work? I had trouble when I first installed them and I had to keep specific versions for them. So can you just validate this is good (doesn't break anything)?

Reply:
yeah it still works